debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/SnapshotReaderIT.java

```diff
@@ -31,12 +36,16 @@
  * @author Randall Hauch
  *
  */
+@RunWith(Arquillian.class)
 public class SnapshotReaderIT {
 
     private static final Path DB_HISTORY_PATH = Testing.Files.createTestingPath("file-db-history-snapshot.txt").toAbsolutePath();
     private static final String DB_NAME = "connector_test_ro";
     private static final String LOGICAL_NAME = "logical_server_name";
 
+    @ArquillianResource
```

Comment:
Unfortunately not a one I'd be aware of. But we can take it a step further and write a provider that would provide an instance of MySQLCube using @ArquillianResource mechanism - in which case we would not need injection of DockerClient